src/particles.cpp

```diff
@@ -243,14 +245,32 @@ void Particle::updateVertices()
 	ParticleSpawner
 */
 
-ParticleSpawner::ParticleSpawner(IGameDef *gamedef, LocalPlayer *player,
-	u16 amount, float time,
-	v3f minpos, v3f maxpos, v3f minvel, v3f maxvel, v3f minacc, v3f maxacc,
-	float minexptime, float maxexptime, float minsize, float maxsize,
-	bool collisiondetection, bool collision_removal, u16 attached_id, bool vertical,
-	video::ITexture *texture, u32 id, const struct TileAnimationParams &anim,
+ParticleSpawner::ParticleSpawner(
+	IGameDef *gamedef,
+	LocalPlayer *player,
+	u16 amount,
+	float time,
+	v3f minpos,
+	v3f maxpos,
+	v3f minvel,
+	v3f maxvel,
+	v3f minacc,
+	v3f maxacc,
```

Comment:
Grouping min&max would ease the overview

Reply:
Ok.

Reply:
Done.